cmd/infra/powervs/create.go

```diff
@@ -986,33 +986,41 @@ func (infra *Infra) createCloudConnection(options *CreateInfraOptions, client *i
 	return
 }
 
-// setupPowerVSDhcp takes care of setting up dhcp in powervs
-func (infra *Infra) setupPowerVSDhcp(options *CreateInfraOptions, session *ibmpisession.IBMPISession) (err error) {
+// useExistingDHCP returns details of existing DHCP server
+func useExistingDHCP(dhcpServersLen int, dhcpServers models.DHCPServers) (*models.DHCPServerDetail, error) {
```

Comment:
I don't see a reason why we need to send dhcpServersLen as a param here, we can also derive the count from dhcpServers

Reply:
Thought of avoiding making len call twice, so passed as an arg.